hack/start.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -x
+
+# Operator startup script, mainly for e2e tests.
+# - Applies everything from manifests/ directory
+# - Prepares operator Deployment based on $IMAGE_FORMAT (if set)
+# - Applies the Deployment
+
+set -e
+
+manifest=$(mktemp -d)
+
+# save generated manifest files in _output directory for debugging purposes
+cleanup(){
+    local return_code="$?"
+
+    set +e
+
+    if [ -n "$ARTIFACT_DIR" ]; then
+        mkdir -p $ARTIFACT_DIR/manifest
+        cp ${manifest}/* $ARTIFACT_DIR/manifest/
+    fi
+
+    rm -rf ${manifest}
+    exit $return_code
+}
+
+trap cleanup exit
+
+if [ -z "${IMAGE_FORMAT}" ] ; then
+    imageBase="registry.svc.ci.openshift.org/ocp/4.5:"
```

Comment:
I think it'd be better to bail or to print print a warning message stating that this is going to be used as base.

In openshift-tests I've had  problems debugging issues because I didn't realized a URL was being defaulted to previous OCP version.

Reply:
I still think it should be possible to run the driver from command line, without $IMAGE_FORMAT. I added a warning around.